encoding/encoding.go

```diff
@@ -38,6 +38,10 @@ const Identity = "identity"
 
 // Compressor is used for compressing and decompressing when sending or
 // receiving messages.
+//
+// If a Compressor implements `DecompressedSize(compressedBytes []byte) int`,
+// gRPC will invoke it to determine the size of the buffer allocated for the
+// result of decompression.  A return value of -1 indicates unknown size.
```

Comment:
I guess this whole package is experimental, so the extra-extra experimental wouldn't make any sense...

Reply:
Yeah, that's why I got rid of it.